CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 ## 1.23.0 (Unreleased)
 
+## 1.22.3 (December 23, 2020)
+
+ENHANCEMENTS:
+* **New Resource:** `opentelekomcloud_obs_bucket_policy` ([#773](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/773))
+* **New Data Source:** `opentelekomcloud_obs_bucket_object` ([#780](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/780))
+* `resource/opentelekomcloud_cce_node_v3`: Add `os` argument ([#778](https://github.com/opentelekomcloud/terraform-provider-opentelekomcloud/issues/778))
```

Comment:
Order pls

Reply:
Moved `New` to `FEATURES`, as it was before v1.199